docker/docker-compose.1804.50.yaml

```diff
@@ -16,9 +16,9 @@ services:
   integration-tests:
     image: swift-nio-http2:18.04-5.0
     environment:
-      - MAX_ALLOCS_ALLOWED_create_client_stream_channel=70010
+      - MAX_ALLOCS_ALLOWED_create_client_stream_channel=72010
```

Comment:
This increase (2 allocs per test run) is for the atomic the child channel has: one for the `Atomic` `class`, and one for the heap allocation for the atomic itself.